packages/e2e-test-utils-playwright/src/editor/index.ts

```diff
@@ -59,6 +59,6 @@ export class Editor {
 	openDocumentSettingsSidebar = openDocumentSettingsSidebar;
 	openPreviewPage = openPreviewPage;
 	saveSiteEditorEntities = saveSiteEditorEntities;
-	selectBlockByClientId = selectBlockByClientId;
+	selectBlocks = selectBlocks;
```

Comment:
I think I accidentally introduced `selectBlockByClientId` in #40815.

I've changed it here so it can multiselect, and the way it used clientIds wasn't very useful, so now uses locators.